deployments/helm-chart/templates/controller-deployment.yaml

```
@@ -161,6 +161,9 @@ spec:
           - -ready-status={{ .Values.controller.readyStatus.enable }}
           - -ready-status-port={{ .Values.controller.readyStatus.port }}
           - -enable-latency-metrics={{ .Values.controller.enableLatencyMetrics }}
+{{- if .Values.controller.extraContainers }}
+      {{ toYaml .Values.controller.extraContainers | nindent 6 }}
```

Same feedback as the with the daemonset yaml 
```suggestion
      extraContainers: {{ toYaml .Values.controller.extraContainers | nindent 8 }}
```